packages/nextjs/src/tools/codegen/import-map.ts

```diff
@@ -315,7 +315,7 @@ export const nextJsMapTemplate = (indexedImportMap: Map<string, ModuleExports>)
           .map(
             (namedExport) => `      { name: '${namedExport.name}', value: ${namedExport.value} }`
           )
-          .join(',\n')
+          .join(',\n') + ','
```

Comment:
```suggestion
          .filter(Boolean)
          .join(',\n') + ','
```
Should we add in `.filter(Boolean)` to remove empty lines when `outputExportEntries()` returns empty strings?

Reply:
`outputExportEntries` should not return empty strings, really